Makefile

```diff
@@ -281,13 +281,13 @@ yosys.html: misc/yosys.html
 
 else ifeq ($(CONFIG),wasi)
 ifeq ($(WASI_SDK),)
-CXX = clang++
+CXX = clang
```

Comment:
Why this change?

Reply:
Because the Yosys build system is [fundamentally broken](https://github.com/YosysHQ/yosys/issues/2011) at the moment. All the other configurations alo have `CXX = <c-compiler-frontend>` in them.

Reply:
Ah, I see this is the default:
https://github.com/YosysHQ/yosys/blob/ff7a8d0e1e1e6fab3e508bb8c237502ef570132b/Makefile#L185